project.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE = html>
+<html lang="en">
+<head>
+  	<meta charset="utf-8">
+  	<title>Portfolio Project</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link href="normalize.css" type="text/css" rel="stylesheet">
+	<link href="Portfoliostyle.css" type="text/css" rel="stylesheet">	
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+	<nav>
+		<ul>
+			<li><a href="portfolio.html">Home</a></li>
+			<li><a href="portfolio.html#portfolio">Portfolio</a></li>
+			<li><a href="portfolio.html#aboutme">About Me</a></li>
+			<li><a href="portfolio.html#contact">Contact</a></li>
+		 </ul>
+		 <!-- <div><img src="" alt=""/> Place for logo -->
+	</nav>
+
+	<div class="container project">
+		<div class="projectimg"><img src="projectimg.jpg" alt="Photos of Project" width="600"></div>
```

Comment:
Just noticed this picture has a small white border to the right and bottom, not sure if thats intentional
![Screenshot 2020-09-08 at 15 09 09](https://user-images.githubusercontent.com/65677163/92487774-9e11e880-f1e5-11ea-93cd-1a4c2da3b6de.png)

Reply:
I'm going to take this page out of the equation for now because it's causing so much trouble! I'll work on it more if I get time.

Reply:
All good no worries